src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -156,7 +154,6 @@
             TokenTypes.SEMI,
             TokenTypes.POST_INC,
             TokenTypes.POST_DEC,
-            TokenTypes.ELLIPSIS,
```

Comment:
we do not need to remove token from Check and will break a lot in this world.

If Google style do not need this token, ti should be removed from Google config.

Reply:
@romani @rnveach I understand, so we don't need change anything about #6707, it can be fixed through user's configuration, right?

Reply:
Yes, at  least  that is  how see issue now.
As you generate diff report we might see some other issue.

Reply:
please revert this change.

Reply:
done